src/candidatelist/service/candidate-list.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { Http, Response } from '@angular/http';
+
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/map';
+
+import { Campaigns } from '../../models/common/campaign.model';
+
+const LEVERAGE_API: string = 'http://api.leveragecampaignfinance.org';
+
+@Injectable()
+export class CandidateListService {
+    constructor(private http: Http) {}
+
+    getCampagins(): Observable<Campaigns[]> {
+        return this.http
+          .get(LEVERAGE_API + '/campaigns')
```

Comment:
Though this will work for now, it works for the wrong reasons. You probably want the [`/candidates`](https://github.com/Lever-age/api/wiki/Interface#candidates) endpoint if you're trying to produce a list of candidates.